backend/nodes/pytorch_nodes.py

```diff
@@ -138,26 +145,27 @@ def __init__(self):
         self.sub = "Processing"
 
     def upscale(self, img: np.ndarray, model: torch.nn.Module, scale: int):
-        # Borrowed from iNNfer
-        logger.info("Converting image to tensor")
-        img_tensor = np2tensor(img, change_range=True)
-        if os.environ["isFp16"] == "True":
-            model = model.half()
-            img_tensor = img_tensor.half()
-        logger.info("Upscaling image")
-        t_out, _ = auto_split_process(
-            img_tensor,
-            model,
-            scale,
-        )
-        del img_tensor, model
-        logger.info("Converting tensor to image")
-        img_out = tensor2np(t_out.detach(), change_range=False, imtype=np.float32)
-        logger.info("Done upscaling")
-        if torch.cuda.is_available():
-            torch.cuda.empty_cache()
-        del t_out
-        return img_out
+        with torch.no_grad():

```

Comment:
It actually shouldn't be necessary with the inference mode decorator I use, but I wanna get rid of that at some point since it makes older torch versions not work